test/client/main.cpp

```diff
@@ -512,15 +494,21 @@ int main(int argc, char* argv[])
                 continue;
             }
             else if (std::strcmp(arg, "--client-reset-metadata-dir") == 0) {
-                if (get_string_value(client_reset_metadata_dir))
-                    continue;
+                std::cerr << "--client-reset-metadata-dir is no longer supported"
+                          << "\n";
+                // if (get_string_value(client_reset_metadata_dir))
+                continue;
             }
             else if (std::strcmp(arg, "--disable-client-reset-recover-local-changes") == 0) {
-                disable_client_reset_recover_local_changes = true;
+                std::cerr << "--disable-client-reset-recover-local-changes is no longer supported"
+                          << "\n";
+                // disable_client_reset_recover_local_changes = true;
                 continue;
             }
             else if (std::strcmp(arg, "--disable-client-reset-require-recent-state-realm") == 0) {
-                disable_client_reset_require_recent_state_realm = true;
+                std::cerr << "--disable-client-reset-require-recent-state-realm is no longer supported"
```

Comment:
Can we just remove this? I think this is just test code.

Reply:
Removed